improver_tests/cube_combiner/test_CubeCombiner.py

```diff
@@ -257,6 +277,22 @@ def test_bounds_expansion(self):
         self.assertEqual(result.coord("time").points[0], 1447894800)
         self.assertArrayEqual(result.coord("time").bounds, [[1447887600, 1447894800]])
 
+    def test_bound_no_expansion(self):
```

Comment:
```suggestion
    def test_bounds_no_expansion(self):
```
Just so that this test matches the previous test `test_bounds_expansion`

Reply:
Done